app/models/manageiq/providers/embedded_terraform/automation_manager/job.rb

```diff
@@ -155,4 +157,13 @@ def cleanup_git_repository
   rescue Errno::ENOENT
     nil
   end
+
+  def input_vars_type_constraints
```

Comment:
This method name doesn't seem to match since it looks like it returns the whole input_vars key, would `template_input_vars` work maybe?

Reply:
Actually I guess this is fine since you're using the template input_vars for type checking